lstchain/io/io.py

```diff
@@ -486,265 +478,6 @@ def write_mcheader(mcheader, output_filename, obs_id=None, filters=HDF5_ZSTD_FIL
         writer.write("run_config", [extramc, mcheader])
 
 
-def read_single_optics(filename, telescope_name):
```

Comment:
weren't some of these functions used anywhere?

Reply:
They were completely superfluous. They were still being used in the `dl1_muon_analysis` script, see changes here:

https://github.com/cta-observatory/cta-lstchain/pull/802/files#diff-86d622a61e69ff5821bd0f9268f4477b7444c34bac6b86c515ff8cb3bdcc8c90

But could just be removed, because the full subarray was already read using `Subarray.from_hdf`